docs/ARCHITECTURE.rst

```diff
@@ -192,10 +192,9 @@ PUT
       versions once versioning has been suspended, which is discussed in
       `"Null Version Management" <#null-version-management>`__.
 
-Only one option is used at a time. ``versionId: <versionId>`` does not
-have to be used with ``versioning: true`` set to work, nor should they
-both be set. If both are used at once, the metadata engine will return
-an error.
+In general, only one option is used at a time. When ``versionId`` and
+``versioning`` are both set, it is for internal needs (e.g. cross-region
```

Comment:
If versionId and versioning is set, then a new version with versionId will be created and master will be updated if the versionId is later than the current master.